src/app/shared/view-issue/new-team-response/new-team-response.component.ts

```diff
@@ -102,7 +102,7 @@ export class NewTeamResponseComponent implements OnInit {
         const [updatedIssue, updatedComment] = resultArr;
       updatedIssue.issueComment = updatedComment;
       this.issueUpdated.emit(updatedIssue);
-      form.resetForm(this.newTeamResponseForm.value);
+      form.resetForm();
```

Comment:
Reverted the changes to reset form. Because ideally, we should revert the new form back to its original state if everything succeeds.